packages/zoe/src/makeHandle.js

```diff
@@ -1,8 +1,29 @@
 // @ts-check
 
 import { assert } from '@agoric/assert';
+import { provide } from '@agoric/store';
+import { defineDurableKind, makeKindHandle } from '@agoric/vat-data';
 import { Far } from '@endo/marshal';
 
+/**
+ * @template {string} H
+ * // ??? Purposeful use of `any`. Should we use `unknown` instead?
+ * @param {MapStore<string, any>} baggage
+ * @param {H} handleType
+ * @returns {() => Handle<H>}
+ */
+export const defineDurableHandle = (baggage, handleType) => {
+  assert.typeof(handleType, 'string', 'handleType must be a string');
+  const durableHandleKindHandle = provide(
+    baggage,
+    `${handleType}KindHandle`,
+    () => makeKindHandle(`${handleType}Handle`),
+  );
+  const makeHandle = defineDurableKind(durableHandleKindHandle, () => ({}), {});
+  return /** @type {() => Handle<H>} */ (makeHandle);
```

Comment:
this doesn't work?
```suggestion
  return makeHandle;
```

Reply:
I get
```
 yarn lint
yarn run v1.22.19
$ run-s --continue-on-error lint:*
$ eslint .
$ tsc --maxNodeModuleJsDepth 4 -p jsconfig.json
src/makeHandle.js:24:3 - error TS2322: Type '() => KindFacet<{}>' is not assignable to type '() => Handle<H>'.
  Type 'KindFacet<{}>' is not assignable to type 'Handle<H>'.
    Type 'KindFacet<{}>' is not assignable to type 'H'.
      'H' could be instantiated with an arbitrary type which could be unrelated to 'KindFacet<{}>'.

24   return makeHandle;
     ~~~~~~~~~~~~~~~~~~


Found 1 error in src/makeHandle.js:24
```

I suspect this has a simple solution, but I do not know what that is.

Reply:
@turadg I'm leaving this conversation open until I hear back.

Reply:
The only simpler solution that comes to mind is a new function like `defineDurableHandle`. The `defineDurableKind` function assumes that the facets object has entries with behaviors and its `KindFacet` wraps those with a context remover.

Alternately we could have the type be conditioned to not use `KindFacet` when the object is empty, but you'd still be getting an empty object and not the `Handle` you're after.

I think the code in the PR is acceptable for now so I'm resolving the comment. If we see this casting repeatedly for making handles it may be worth a new function designed for this use case.